Update serilog instruction (#4365)
DataDog/documentation: content/en/logs/log_collection/csharp.md

{{< tabs >}}
{{% tab "SeriLog" %}}

> Install the Datadog [Serilog sink][1], which send events and logs to Datadog. By default the sink forwards logs through HTTPS on port 443.

Reviewer: s/send/sends/

    .CreateLogger();
```

> Logs can be sent to the EU site by setting the `url` property to `https://http-intake.logs.datadoghq.eu`

Reviewer 2: ```suggestion
**Note**: To send logs to Datadog EU site, set the `url` property to `https://http-intake.logs.datadoghq.eu`

```

> You can also override the default behavior and forward logs in TCP by manually specifying the following properties in the parameters: `url`, `port`, `useSSL` and `useTCP`. You can also specify the `source`, `service`, `host`, and custom tags:

Reviewer 2: ```suggestion
You can also override the default behaviour and forward logs in TCP by manually specifying the following required properties: `url`, `port`, `useSSL` and `useTCP`. Optionally, specify the `source`, `service`, `host`, and custom tags.
For instance to forward logs to the Datadog US site in TCP you would use the following sink configuration :
```

    .CreateLogger();
```

**Note**: To send logs to Datadog EU site, set the `url` property to `https://http-intake.logs.datadoghq.eu`

> You can also override the default behaviour and forward logs in TCP by manually specifying the following required properties: `url`, `port`, `useSSL` and `useTCP`. Optionally, specify the `source`, `service`, `host`, and custom tags.

Reviewer 2: ```suggestion
You can also override the default behaviour and forward logs in TCP by manually specifying the following required properties: `url`, `port`, `useSSL`, and `useTCP`. Optionally, specify the `source`, `service`, `host`, and custom tags.
```

> You can also override the default behaviour and forward logs in TCP by manually specifying the following required properties: `url`, `port`, `useSSL`, and `useTCP`. Optionally, specify the `source`, `service`, `host`, and custom tags.

Reviewer: Should these options link somewhere to define what they do and their format? This sentence sounds a bit odd as is as well.
